easybuild/easyblocks/a/abaqus.py

```diff
@@ -255,7 +253,7 @@ def install_step(self):
                         raise EasyBuildError("Failed to find expected subdir for hotfix: %s", subdirs)
 
                     cwd = change_dir(os.path.join(cwd, subdir, '1'))
-                    run_cmd_qa('./StartTUI.sh', {}, std_qa=std_qa, log_all=True, simple=True, maxhits=100)
+                    run_shell_cmd('./StartTUI.sh', qa_patterns=qa, qa_wait_patterns=no_qa, qa_timeout=100)
```

Comment:
```suggestion
                    run_shell_cmd('./StartTUI.sh', qa_patterns=qa, qa_wait_patterns=no_qa, qa_timeout=200)
```

We may need to make this longer.